config/jobs/kubernetes-security/genjobs.go

```diff
@@ -457,20 +489,22 @@ func main() {
 	}
 	defer os.Remove(f.Name())
 
-	// write the original bytes before the security repo section
-	_, err = f.Write(originalBytes[:securityRepoStart])
-	if err != nil {
-		log.Fatalf("Failed to write temp file: %v", err)
-	}
-	f.Sync()
-	io.WriteString(f, "\n")
+	// write the header
+	io.WriteString(f, "# Autogenerated by genjobs.go please do not edit!\n")
+	io.WriteString(f, "# To modify these jobs first update their kubernetes/kubernetes sources\n")
```

Comment:
nit: kubernetes/test-infra sources?

Reply:
can I also cover this in the same follow-up? will re-word.